releasenotes/notes/remove-forwarder_retry_queue_max_size-a4b473b1249081f4.yaml

```diff
@@ -0,0 +1,17 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    `forwarder_retry_queue_max_size` is deprecated in favor of 
+    `forwarder_retry_queue_payloads_max_size`. If 
+    `forwarder_retry_queue_max_size` is not set, you are not affected by this 
+    change. If `forwarder_retry_queue_max_size` is set but 
+    `forwarder_retry_queue_payloads_max_size` is not set, the Agent displays a
+    warning telling that `forwarder_retry_queue_max_size * 2MB` is used as the  
+    value of `forwarder_retry_queue_payloads_max_size`.
```

Comment:
```suggestion
    If `forwarder_retry_queue_max_size` is set but 
    `forwarder_retry_queue_payloads_max_size` is not set, the Agent will use  `forwarder_retry_queue_max_size * 2MB` 
    as the  value of `forwarder_retry_queue_payloads_max_size`.
```

Reply:
Release note updated. (Your commit suggestion remove `change` from the previous sentence).